### PR TITLE
Add ikea

### PR DESCRIPTION
**Pull request by stephengroat**
in honor of college starting in the US, let's secure the college furniture store of the US

### CODE REVIEW

**Comment by Carlgo11** [comment]
*cough\* but IKEA is Swedish 😉

**Comment by stephengroat** [comment]
touché

IKEA is really a celebration of cheap people everywhere, not just US college students. Obviously a nationalistic blunder on my part :)

File: _data/retail.yml [hunk 1]
```diff
@@ -103,6 +103,13 @@ websites:
       twitter: hypop
       tfa: No
 
+    - name: IKEA
+      url: http://www.ikea.com
+      img: ikea.png
+      tfa: No
+      twitter: IKEA
```
Comment (Carlgo11) [inline _data/retail.yml]: How about IKEAUSA here as well? They seem to actually use that account.

**Comment by Carlgo11** [comment]
I feel like it's my duty as a Swedish citizen to provide a better image to use.
Here have this one: 
IKEA  
<br>  
But, well... this image also works I suppose...
För kung och fosterland

**Comment by stephengroat** [comment]
Good catch, I just checked to make sure that the account existed.

And if you can shrink it down, maybe it should be made the tooltip for ikea, just to remind everyone about sweeden's greatness

**Comment by Carlgo11** [comment]
That could work. Or we could change the entire page's background to the great Swedish flag and an `<audio>` tag to play the national anthem.

**Pull request merged**